src/__tests__/index.js

```diff
@@ -169,7 +169,7 @@ import Demo from './demo'
   assert.equal(
     error.message,
     `Build failed with 1 error:
-__mdx_bundler_fake_dir__/demo.tsx:1:7: error: [plugin: inMemory] Could not resolve "./blah-blah" from ".${path.sep}demo.tsx"`,
+__mdx_bundler_fake_dir__/demo.tsx:1:7: error: Could not resolve "./blah-blah"`,
```

Comment:
It's a shame that it doesn't tell us the importer anymore, but I think I can live with that for this. Thanks!